lib/tapioca/compilers/dsl/active_record_columns.rb

```diff
@@ -90,7 +78,7 @@ module Dsl
       #     sig { params(value: ::String).returns(::String) }
       #     def title=(value); end
       #
-      #     sig { params(args: T.untyped).returns(T::Boolean) }
+      #     sig { returns(T::Boolean) }
```

Comment:
Late to the battle but is this one correct?

Should it be `title?; end`?

Reply:
Thank you. Fixed in https://github.com/Shopify/tapioca/commit/16bb1be8947e60ca08b8eba6b929e695e18cd7f8